spec/services/data_migrations/fix_misspelling_of_caribbean_ethnic_group_and_set_hesa_codes_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe DataMigrations::FixMisspellingOfCaribbeanEthnicGroupAndSetHesaCodes do
```

Comment:
Could we pad this spec out with tests for `hesa_ethnicity` already being set and different recruitment cycle years (not sure if that actually makes a difference)